File: matiasmasso/Mat.NET/DT2/DTOPagination.vb
Public Class DTOPagination
    Property ItemsCount As Integer
    Property PageSize As Integer
    Property PageIndex As Integer
    Property Guid As Guid

    Public Sub New(iItemsCount As Integer, iPageSize As Integer, iPageIndex As Integer, Optional oGuid As Guid = Nothing)
        MyBase.New()
        _ItemsCount = iItemsCount
        _PageSize = iPageSize
        _PageIndex = iPageIndex
        _Guid = oGuid
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOPagination.cs
using System;

namespace DTO
{
    public class DTOPagination
    {
        public int ItemsCount { get; set; }
        public int PageSize { get; set; }
        public int PageIndex { get; set; }
        public Guid Guid { get; set; }

        public DTOPagination(int iItemsCount, int iPageSize, int iPageIndex, Guid oGuid = default(Guid)) : base()
        {
            ItemsCount = iItemsCount;
            PageSize = iPageSize;
            PageIndex = iPageIndex;
            Guid = oGuid;
        }
    }
}
